src/cmds/init.py

```diff
@@ -44,6 +44,14 @@ def migrate_from(old_root, new_root, manifest):
     # and make what may have been relative paths absolute
 
     plots_config = load_config(new_root, "plots.yaml")
+
+    new_plots_root = path_from_root(
+        old_root,
+        load_config(new_root, "config.yaml")
+        .get("harvester", {})
+        .get("plot_root", "plots"),
+    )
```

Comment:
You need to change `plots` to `.` because that's the default value that's used in the harvester.